### PR TITLE
[BH-1542] Fix TimeFixedWidget encapsulation

### PR DESCRIPTION
**Pull request by disonek**
**Description**

**Your checklist for this pull request**

Make sure that this PR:
- [ ] Complies with our guidelines for contributions
- [x] Has unit tests if possible.
- [ ] Has documentation updated

### CODE REVIEW

File: module-apps/apps-common/widgets/DigitsContainer.hpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        std::uint32_t colonWidth;
+        std::uint32_t minusWidth;
+        std::uint32_t leftBoxWidth;
+        std::uint32_t rightBoxWidth;
+    };
+
+    template <size_t N> struct DigitsContainer
+    {
+        std::array<Label *, N> digits;
+        void setMinutesBox(uint32_t minutes, DimensionsParams params);
```
Comment (swichu91) [inline module-apps/apps-common/widgets/DigitsContainer.hpp:24]:
```suggestion
        void setMinutesBox(std::uint32_t minutes, DimensionsParams params);
```

File: module-apps/apps-common/widgets/DigitsContainer.hpp [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        void setSecondsBox(uint32_t seconds, DimensionsParams params);
```
Comment (swichu91) [inline module-apps/apps-common/widgets/DigitsContainer.hpp:25]:
```suggestion
        void setSecondsBox(std::uint32_t seconds, DimensionsParams params);
```

File: module-apps/tests/widgets/test-TimeFixedWidget.cpp [hunk 3]
```diff
@@ -43,35 +29,31 @@ namespace
...
 
-TEST_F(TimeFixedWidgetTest, minusNotVisable)
-{
-    widget->setFontAndDimensions(style::window::font::verybig);
-    EXPECT_EQ(widget->getLeftBox().minus->getText(), "");
-}
+} // namespace
 
-TEST(TimeFixedWidgetNonFixtureTest, minusVisable)
+TEST(TimeFixedWidgetTest, initilization)
```
Comment (swichu91) [inline module-apps/tests/widgets/test-TimeFixedWidget.cpp:44]: I'd remove this test case. Additionally, I'd rename the whole `test-TimeFixexWidget` to `test-DigitsContainer`.

File: module-apps/apps-common/widgets/TimeFixedWidget.hpp [hunk 4]
```diff
@@ -3,40 +3,29 @@
...
     {
       public:
-        template <size_t N> using DigitsContainer = std::array<Label *, N>;
-
         struct LeftBox
         {
             HBox *box    = nullptr;
             Label *minus = nullptr;
-            DigitsContainer<3> digits{nullptr};
+            DigitsContainer<3> container{nullptr};
```
Comment (swichu91) [inline module-apps/apps-common/widgets/TimeFixedWidget.hpp:22]: Initialising it with `nullptr` is a bit misleading here for me. Imho, it'd be better to use default constructor if you want to do such things as this is definitely not a proper way to initialise `std::array<Label *, N>`

File: module-apps/apps-common/widgets/TimeFixedWidget.hpp [hunk 5]
```diff
@@ -3,40 +3,29 @@
...
         };
 
         struct RightBox
         {
             HBox *box = nullptr;
-            DigitsContainer<2> digits{nullptr};
+            DigitsContainer<2> container{nullptr};
```
Comment (swichu91) [inline module-apps/apps-common/widgets/TimeFixedWidget.hpp:28]: Same here ;)

**Review by swichu91 — APPROVED** [review]

**Review by dawidwojtas-mudita — APPROVED** [review]

**Pull request merged**